src/mp-instance.ts

```diff
@@ -66,6 +67,7 @@ export type IntegrationDelays = Dictionary<boolean>;
 // https://go.mparticle.com/work/SQDSDKS-6949
 export interface IMParticleWebSDKInstance extends MParticleWebSDK {
     // Private Properties
+    _timer: ForegroundTimer;
```

Comment:
`_timer` is too generic. I would call this something like `_foregroundTimer`, `_timeOnSiteTimer` or something relevant to the use case.

Reply:
good call.  this was originally just a filler to get a POC going.    i'll update now